requirements.txt

```diff
@@ -35,6 +35,7 @@ requests==2.23.0
 retrying==1.3.3
 six==1.14.0
 soupsieve==2.0
+spotipy==2.12.0
```

Comment:
Let's not forget to add new packages to requirements. `pip install PACKAGE && pip freeze > requirements.txt`